### PR TITLE
No decoration

### PR DESCRIPTION
**Pull request by rofe**
Prep for https://github.com/adobe/helix-pages/issues/403 (see [here](https://github.com/adobe/helix-pages/pull/416#issuecomment-674092761) for instructions how to test it)

@kptdobe I'm afraid this is currently breaking banners (or internal embeds in general) again, as their CSS seem to rely on a `default` class on the child divs. It would be great if you could throw an expert eye on it!

### CODE REVIEW

**Comment by rofe** [comment]
Note: these changes also work fine with the current version of Helix Pages.

**Comment by kptdobe** [comment]
I think for embed, it is easy, `default` is I think useless anyway, the `div` is enough. But many other places rely on the `default` class: https://github.com/adobe/theblog/search?l=CSS&q=default

**Comment by kptdobe** [comment]
`image` also: https://github.com/adobe/theblog/blob/d443082c2d322ae9c04b5322e618589109f2e4de/style.css#L39-L47

**Comment by rofe** [comment]
> I think for embed, it is easy, `default` is I think useless anyway, the `div` is enough. But many other places rely on the `default` class: https://github.com/adobe/theblog/search?l=CSS&q=default

Yes, I know, I patched all of them in this PR except for [these 2 in post.css](https://github.com/adobe/theblog/blob/001f5f3ed790f91d6a5affa2a2d1cb5e6dae7e66/style/post.css#L325-L339). To be honest, I didn't know how, I guess this will require adding a wrapper div in JS. It would be great if you could take a look since you're more familiar with this particular feature.

**Comment by rofe** [comment]
> `image` also:

Removed in this PR. As far as I could tell, they weren't used by any code and removing them didn't affect the output.

**Comment by kptdobe** [comment]
oh sorry, for some reason, I though we were in helix-pages :)
I used those 2 articles for testing the new and old banner styles: 
https://no-decoration--theblog--adobe.hlx.page/en/publish/2020/08/13/smb-playbook-for-customer-experience-management.html#gs.dob458
https://no-decoration--theblog--adobe.hlx.page/en/publish/2020/08/13/quick-mockup-2-0-templates-themes.html#gs.dob168
But for some reasons, the new ones are not styled properly without any changes. Need to look deeper.

**Comment by kptdobe** [comment]
Ah.. your branch was far behind master. I just merged master, you might need to retest your changes :)

File: style.css [hunk 1]
```diff
@@ -32,25 +32,6 @@
...
-   margin: 0;
-   font-size: 0;
- }
- 
- .image img {
-   width: 100%;
- }
- 
- .default img {
-   max-width: 100%;
```
Comment (kptdobe) [inline style.css:50]: This one has a big impact: all content images had a `max-width: 100%` before that is not there anymore.

Comment (rofe) [reply style.css:50]: Ok, but it only affected images within embeds, right? I couldn't spot a difference in rendering of normal article images.

Comment (kptdobe) [reply style.css:50]: Yes. I just raised the concern, there might be some cases we do not think of...

**Comment by rofe** [comment]
@kptdobe thanks for fixing the banners!

Did you also test locally with the `no-decoration` branch of `helix-pages` (see [here](https://github.com/adobe/helix-pages/pull/416#issuecomment-674092761)

I just re-checked all page types side by side and to me the output looks identical.

**Comment by kptdobe** [comment]
No, I used to dev tools to simply remove the `default` class inside the embeds. But i can make the test following the instructions.

**Comment by kptdobe** [comment]
which was a good idea, I found something else broken... investigating...

**Comment by kptdobe** [comment]
Ah.. now the embed is not wrapped inside a div. Need more changes.

**Comment by rofe** [comment]
Continued in #422

**Pull request closed without merging**